src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,383 @@
+            </td>
+            <td>
+                The path to the resource for requesting an access token
+            </td>
+            <td>
+                "/access_token"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>"preferencesConfig.securityServer.paths.preferences"</code>
+            </td>
+            <td>
+                The path to the resource for creating a preference set.
+                A query parameter can be added to provide the view/ontology that the
+                preferences are to be stored in.
+            </td>
+            <td>
+                "/add-preferences?view=%view"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>"preferencesConfig.authentication.grant_type"</code>
+            </td>
+            <td>
+                The grant type supported by the OAuth2 server.
+            </td>
+            <td>
+                "client_credentials"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>"preferencesConfig.authentication.scope"</code>
+            </td>
+            <td>
+                The level of permissions that are being requested.
+            </td>
+            <td>
+                "add_preferences"
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### vagrant.json ###
+
+Builds on [environment.json](https://github.com/GPII/first-discovery-server/tree/master/src/config/environment.json) and [oauth2.json](https://github.com/GPII/first-discovery-server/tree/master/src/config/oauth2.json), setting concrete values to configuration of the OAuth2 server that are expected when being run in a  [Vagrant VM](https://www.vagrantup.com). The configuration assumes that an instance of the OAuth2 server is available through the host machine at port 8081.
+
+A Vagrant VM is provided with the First Discovery Server for quickly creating its own [development environment](https://github.com/GPII/first-discovery-server/blob/master/README.md#development).
+
+(see: [vagrant.json](https://github.com/GPII/first-discovery-server/tree/master/src/config/vagrant.json))
+
+<table>
+    <thead>
+        <tr>
+            <th>Option Path</th>
+            <th>Description</th>
+            <th>Value</th>
+        <tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <code>"preferencesConfig.securityServer.port"</code>
+            </td>
+            <td>
+                The port number that the GPII OAuth2 server is hosted at.
+            </td>
+            <td>
+                "8081"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>"preferencesConfig.securityServer.hostname"</code>
+            </td>
+            <td>
+                The hostname that the GPII OAuth2 server is hosted at.
+            </td>
+            <td>
+                "http://10.0.2.2"
```

Comment:
It looks like this variable actually ends up at a member named "url" which can hold an entire URL. It would be more accurate to rename this to some name such as "url" since it doesn't hold a hostname as its name suggests.